_data/projects/H.yml

```diff
@@ -0,0 +1,11 @@
+name: H
+desc: Html templating library written in kotlin
+site: https://github.com/phenax/h
+tags:
+- html
+- kotlin
+- template
+- engine
+upforgrabs:
+  name: help wanted
+  link: https://github.com/phenax/h/issues/1
```

Comment:
This should be a link to a list of issues, rather than a single one, e.g: https://github.com/phenax/h/labels/help%20wanted